wiki/faq.md

```diff
@@ -242,17 +242,129 @@ When you stake on a [fork](forks "wikilink"), the stake you see is not actually
 If you see your pending rewards at 16384 GRC, you have hit the maximum amount of
 Gridcoin that can be sent per stake. When you stake you will only earn 16384 GRC
 and any amount of extra work done will not be included. If you keep hitting this
-limit, make sure you are running your wallet to stake 24/7 (or close to it). If you are, 
-you should get more Gridcoin to stake more frequently. 
+limit, make sure you are running your wallet to stake 24/7 (or close to it). If you are,
+you should get more Gridcoin to stake more frequently.
 
-If you want to get an estimate of how likely you are to stake *before* that limit, you can try solving 
+If you want to get an estimate of how likely you are to stake *before* that limit, you can try solving
 for that probability using:
 `16384/(magnitude/4) = -ln( 1 - Probability ) * 10000 / Balance * Difficulty`.  
 The probability there are the odds that you stake at least once at any point before that limit (it's a confidence interval).
 
 Note: It is highly unlikely for most users to run into this limit. It requires a very
 high magnitude to possibly run into this.
 
+---
+# Manual Reward Claim
+
+---
+### What is Manual Reward Claim (MRC)?
+
+Manual Reward Claim (MRC) is a feature in the official Gridcoin wallet [release version 5.4.0.0](https://github.com/gridcoin-community/Gridcoin-Research/releases/tag/5.4.0.0).
+When you earn BOINC credits in solo crunching mode, you will soon accumulate
+GRC that is owed to you. This GRC doesn't exist yet and you can claim it via two
+methods, staking or MRC.
+
+MRC allows a user to claim their owed GRC without the requirement of having a
+large amount of GRC to stake it (like in previous versions).
+
+MRC was made to solve the issue of new users being unable to claim rewards
+without either acquiring GRC first or using a pool.
+
+---
+### If I have enough GRC to stake, do I need to use MRC?
+
+No. MRC is optional. If you would like to advance the payment of your owed GRC
+without having to wait for your next stake, then you are able to issue a MRC
+request from the wallet.
+
+---
+### How often can I make a MRC request?
+
+Once every 14 days is the most often you can make a MRC request.
+Note that if you have only just created your Gridcoin wallet and advertised your
+beacon, you will need to wait 14 days until your first request.
+

```

Comment:
I would mention that you might not want to claim this often because of the high fees compared to waiting